schema/common.json

```diff
@@ -1,7 +1,6 @@
 {
     "$id": "https://opentelemetry.io/otelconfig/common.json",
     "$schema": "https://json-schema.org/draft/2020-12/schema",
-    "title": "Common",
```

Comment:
Notably, removing `title` in top level schema like this, which collect types for re-use but which aren't referenced anywhere in the schema directly tidies up the result of code generation. 

Previously, the `opentelemetry-java` implementation generated a `CommonModel` type. It wasn't used anywhere, but it still was generated. With this change, its goes away as it should. I could probably find some way to explicitly omit generation for these kind of types, but it is a nice side affect of what seems like a sensible change for other reasons.